docs/index.md

```diff
@@ -0,0 +1,56 @@
+# Welcome!
+
+Welcome to yalu102's documentation. Visit the GitHub Project [here](https://github.com/kpwn/yalu102/)
+
+## About
+
+This is a "work in progress" Jailbreak designed for pre-i7 devices running iOS 10.2.
+
+It was created by [qwertyoruiopz](https://twitter.com/qwertyoruiopz) and [marcograssi](https://github.com/kpwn/yalu102/blob/master/marcograss).
+
+## Support List
+
+Device | iOS | Yalu
+:----------- |:-------------:| -----------:
+iPhone 7+ | 10 - 10.1.1 | Yalu B3
+iPhone 7 | 10 - 10.1.1 | Yalu B3
+iPhone SE | 10 - 10.2 | yalu102
+iPhone 6s+ | 10 - 10.2 | yalu102
+iPhone 6s | 10 - 10.2 | yalu102
+iPhone 6+ | 10 - 10.2 | Not Supported
+iPhone 6 | 10 - 10.2 | Not Supported
+iPhone 5s | 10 - 10.2 | Not Supported
+iPod 6G | 10 - 10.2 | Not Supported
+iPad Pro | 10 - 10.2 | Not Supported
+iPad Air 2 | 10 - 10.2 | Not Supported
+iPad Air | 10 - 10.2 | Not Supported
+iPad Mini 4 | 10 - 10.2 | Not Supported
+iPad Mini 3 | 10 - 10.2 | Not Supported
+iPad Mini 2 | 10 - 10.2 | Not Supported
+
+## Installation
+
+1. Plug in your supported iDevice
+2. Open up Cydia Impactor
+3. Drag and drop the corresponding IPA into Cydia Impactor's window
+4. Sign in with your AppleID
+5. Sideload the IPA
+
+Version | Download | SHA1
+:----------- |:-------------:| -----------:
+yalu103 - Alpha | [Link](https://yalu.qwertyoruiop.com/yalu102_alpha.ipa) | 2fe14f1c1e1a0d26203bbb123f6747a978dd2b4f
+YaluB3 | [Link](https://yalu.qwertyoruiop.com/mach_portal+yalu-b3.ipa) | f34ba1bebf02b265a871158e778596ee2ff1547a69fdb687d911897b7b7b22c5
+
+## Guides
+
+* [Checking your SHA1](sha1/)
+
+## Public Wiki
+
+These docs were brought to you by [JailbreakHub](redirect/)
+
+## Have a problem or suggestion?
+
+* Make a [Pull Request](https://github.com/iOptimistic/yalu102/pulls)
+
+* Contact the [WebMaster](mailto:bobby@jailbreakhub.org?Subject=Yalu102%20Suggestions)
```

Comment:
There is no point in mentioning the site. as Null said, just create a Issue report